modules/javafx.media/src/main/java/com/sun/media/jfxmediaimpl/NativeMediaAudioClipPlayer.java

```diff
@@ -451,6 +451,11 @@ public boolean equals(Object that) {
         }
     }
 
+    @Override
+    public int hashCode() {
+        return sourceClip.hashCode();
+    }
```

Comment:
Why are you ignoring the significant fields? `equals` compares `priority`, `loopCount`, `volume`, `balance`, `rate` and `pan` in addition.